src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -75,6 +92,10 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
         if (iframeRef.current) {
           setSupportedFeatures(iframeRef.current, features);
         }
+        if (phoneRef.current && iframeRef && pluginInfo.textDecorationHandlerInfo) {
```

Comment:
```suggestion
        if (phoneRef.current && pluginInfo.textDecorationHandlerInfo) {
```
I thought I commented on this before, but I don't think this `iframeRef` test is accomplishing anything here.

Reply:
Ah, yes.  Fixed.